knowhere/knowhere/index/vector_index/impl/bruteforce/distances/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
-set (CMAKE_CXX_STANDARD 11)
+set (CMAKE_CXX_STANDARD 17)
```

Comment:
guarantee CXX version is compatible with multiple platforms.

Reply:
ditto.